spec/models/motorcycle_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'rails_helper'
+
+RSpec.describe Motorcycle, type: :model do
+  before(:each) do
+    @motorcycle = Motorcycle.create(:motorcycle)
+  end
+
+  context 'Motorcycle model, ' do
+    it 'should have valid attributes' do
+      expect(@motorcycle).to be_valid
+    end
```

Comment:
[OPTIONAL]
Should always have positive and negative tests to ensure it works the way you expect it. Eg. What do you expect the output to be when it's not valid or when valid etc.
Consider making all your tests more extensive, not just this one. For this simple application, it may not catch a possible crippling bug, but in bigger applications, it definitely can. Consider this for all projects moving forward.

Reply:
Hello @mcrd25 thanks for the review I have made all test cases pass correctly
![CaptureXXIX](https://user-images.githubusercontent.com/83566118/180300670-212fd1be-9653-4a95-ac13-b3e382406428.PNG)
![CaptureXXIV](https://user-images.githubusercontent.com/83566118/180300682-80f7dd59-de23-4409-958a-c8577ea78dcf.PNG)